test/unit/models/forms/v2/MiscDetailsFormSpec.scala

```diff
@@ -98,7 +98,7 @@ class MiscDetailsFormSpec extends ModelsBaseSpec {
         val form = MiscDetailsForm.miscDetailsForm(miscCase)
 
         form.hasErrors shouldBe false
-        form.data      shouldBe params.mapValues(v => v.head)
+        form.data      shouldBe params.map { case (fst, snd) => fst -> snd.head }
```

Comment:
Same with the others as it can just be `params.view.mapValues(v => v.head).toMap`. Thanks